Automattic/wp-calypso
# My Home: Combine photo library buttons to one support dialog and update image.

For the Free Photos card, we currently have an image that opens a dialog with just the video, and a Learn More button that opens a dialog displaying the support page related to free photos. I've added the video to the [support page](https://wordpress.com/support/free-photo-library/), which now means the video displays at the top of the support dialog – kinda making the separate dialog for the video redundant. This PR has clicking on the image open the same support dialog as the Learn More button, meaning whichever the user clicks on, they'll get both sources of support (with the video being first in sight).

This will also make the (not autoplay) video available to users on mobile, since the card image is not displayed at smaller screen widths (only the Learn More button).

<img width="1056" alt="Screen Shot 2020-03-18 at 11 47 19 AM" src="https://user-images.githubusercontent.com/349751/76996170-479ab000-690e-11ea-84f2-261378068775.png">

#### Testing instructions

* On this branch, go to `/home/:site`.
* Verify that clicking the image in the Free Photos card opens the support dialog, and that the Learn More button continues to do the same. The video should be the first content in the support dialog, after the page title.
